ocaml/xapi/xapi_pool.ml

```diff
@@ -2130,8 +2130,12 @@ let send_test_post = Remote_requests.send_test_post
 
 let certificate_install = Certificates.(pool_install CA_Certificate)
```

Comment:
Since they are not API-facing function, I'd like the previous functions to be removed and their usages to be changed to the new ones, same with `xapi_host`

Reply:
These cannot be removed  because of how the API definitions and these module interact.